core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -944,6 +944,7 @@ protected Jenkins(File root, ServletContext context, PluginManager pluginManager
 
 
             if(KILL_AFTER_LOAD)
+                // TODO cleanUp?
```

Comment:
Of course the flag is not for production code, but perhaps users of this flag for whatever purpose expect cleanup tasks to run? I do not know.